library/ui/src/main/java/com/google/android/exoplayer2/ui/DefaultTimeBar.java

```diff
@@ -835,4 +843,8 @@ public static int getDefaultPlayedAdMarkerColor(int adMarkerColor) {
   private static int dpToPx(DisplayMetrics displayMetrics, int dps) {
     return (int) (dps * displayMetrics.density + 0.5f);
   }
+
+  private static int pxToDp(int densityDpi, int px) {
+    return (int) (px / ((float) densityDpi / DisplayMetrics.DENSITY_DEFAULT));
```

Comment:
Isn't (float) densityDpi / DisplayMetrics.DENSITY_DEFAULT the same as displayMetrics.density? See method dpToPx avove. If so, it's probably better to use the same conversion such that both methods obviously do the same thing, just the other way round.

Reply:
Yes, it's the same, I can use that.

Reply:
But I can't pass the DisplayMetrics instead of density, because the width of the view can't be obtained yet in the ctr.